kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -164,6 +171,26 @@ def __init__(self):
+        """Truncate bridge name to fit Linux IFNAMSIZ (max 15 chars).
+
+        If the original name exceeds 15 characters, it is truncated to 9
+        characters followed by a dash and a 5-character hash suffix derived
+        from the full name so that different long names produce different
+        short names.  For example:
+            'hsp-zcf1500-retest'  ->  'hsp-zcf15-e1a2b'
+        """
+        if len(name) <= OvsDesiredState._MAX_IFNAME_LEN:
+            return name
+        import hashlib
+        suffix = hashlib.md5(name.encode('utf-8')).hexdigest()[:5]
```

Ruff (0.15.6)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)